examples/pytorch/mnist/README.md

```diff
@@ -3,20 +3,6 @@
 The document describes how to use DLRover to train a Pytorch CNN model
 with MNIST dataset.
 
-## Prepare Data
```

Comment:
It is better to retain the introduction and add a way to create the dataset by `datasets.MNIST()`.